File: Assg1/Prog1.java
import java.util.Scanner;

public class Prog1{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter number: ");
        int num = sc.nextInt();

        if(isPrime(num))System.out.println(num + " is prime.");
        else System.out.println(num + " is not prime.");
        
    }
     public static boolean isPrime(int num){
        if(num==1 && num ==0) return false;

        boolean prime = true;
        if(num==2) return prime;        

        for (int i = 2; i < num; i++) {
            if(num%i == 0) {
                prime = false;
                break;
            }
            
        }
        return prime;
     }
}

File: AssgJS/Prog1.js
//install: npm install prompt-sync
const prompt=require("prompt-sync")({sigint:true});

let num = prompt("Enter Number: ")
let prime = false;

for(let i = 2; i <= num; i++){
    if(num == i) {
        prime = true;
        break;
    }
    
    if (num%i == 0){
        prime = false;
        
        break;
    }
    
}

prime ? console.log( num + ' is prime') : console.log(num +' not prime');